### PR TITLE
add flaky-test-reporter's config.yaml to its image

### PR DESCRIPTION
**Pull request by TrevorFarrelly**
**What this PR does, why we need it**:
add the reporter's new yaml config format to its Dockerfile in the expected default location.

If it makes more sense somewhere else in the image, we will need to add a flag in `ci/prow/periodic_config.go`.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes to reviewers**:

**User-visible changes in this PR**:

### CODE REVIEW

File: images/flaky-test-reporter/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ ADD . $TEMP_REPO_DIR
 # Build flaky test reporter tool in the container
 RUN make -C $TEMP_REPO_DIR/tools/$TOOL_NAME/
 RUN cp $TEMP_REPO_DIR/tools/$TOOL_NAME/$TOOL_NAME /$TOOL_NAME
+RUN cp $TEMP_REPO_DIR/tools/$TOOL_NAME/config.yaml config.yaml
```
Comment (srinivashegde86) [inline images/flaky-test-reporter/Dockerfile]: why dont we just copy everything under $TOOL_NAME?

Comment (TrevorFarrelly) [reply images/flaky-test-reporter/Dockerfile]: That would add source files, the README, and other things right? Certainly doable, it adds a bit of bloat though.

Comment (srinivashegde86) [reply images/flaky-test-reporter/Dockerfile]: yes. but its just a few files. Other images add all the files in the repo :)

I am fine either way, but seems like everytime you want something in the image, you will have to keep adding it.

**Pull request merged**